cert-tools/snapstore/src/snapstore/cli.py

```diff
@@ -0,0 +1,124 @@
+from argparse import ArgumentParser, Namespace
+import json
+from typing import List
+
+from snapstore.craft import create_base_client
+from snapstore.client import SnapstoreClient
+from snapstore.info import SnapstoreInfo
+from snapstore.snaps import SnapSpecifier, SnapChannel
+
+
+def get_info_arguments(args: List[str] | None = None) -> Namespace:
+    parser = ArgumentParser(
+        description='Retrieve snap info from the Store'
+    )
+    parser.add_argument("snap", type=str)
+    parser.add_argument("channel", type=str)
+    parser.add_argument("architecture", type=str)
+    parser.add_argument("--store", type=str)
+    parser.add_argument(
+        "--fields", nargs="+",
+        help=(
+            "fields to include in the response "
+            "(see https://api.snapcraft.io/docs/refresh.html "
+            "or https://api.snapcraft.io/docs/info.html)"
+        )
+    )
+    parser.add_argument(
+        "--use-info", dest="refresh", action="store_false",
+        help="use `v2/snaps/info` endpoint (default is `v2/snaps/refresh`)"
+    )
+    parser.add_argument(
+        "--token-environment-variable",
+        dest="variable",
+        type=str,
+        default="UBUNTU_STORE_AUTH",
+        help="Variable containing token returned by `snapcraft export-login`",
+    )
+    return parser.parse_args(args)
+
+
+def get_snap_info(info: SnapstoreInfo, args: Namespace) -> dict:
```

Comment:
Ough, why the Namespace here? Can you mark them as internal if you don't want them to be used outside of the `cli` module?